templates/agent/keymanager/keymanager.go

```diff
@@ -125,8 +115,23 @@ func (p *Plugin) SignData(ctx context.Context, req *keymanagerv1.SignDataRequest
 	return nil, status.Error(codes.Unimplemented, "not implemented")
 }
 
+// SetLogger is called by the framework when the plugin is loaded and provides
```

Comment:
IMO, these methods aren't related to plugin/service/hostservice implementations and should either go above (as they are now) or below all of the plugin/service/hostservice methods, not in between the plugin impl and config service impl.

Reply:
Sure, it makes sense. I would go for moving this method below all of the plugin/service/hostservice implementations. I think that having the plugin implementation methods on the top makes more sense since they are the only required methods for implementing the plugin, all the rest is optional.

Reply:
[b1aedd0](https://github.com/spiffe/spire-plugin-sdk/pull/47/commits/b1aedd0ffadea2091feebbb031c9826014dd335d)